core/class_db.cpp

```diff
@@ -1042,6 +1046,7 @@ bool ClassDB::get_property(Object *p_object, const StringName &p_property, Varia
 	while (check) {
 		const PropertySetGet *psg = check->property_setget.getptr(p_property);
 		if (psg) {
+			TEST_PROPERTY_DEPRECATED(check->name, p_property);
```

Comment:
I think this and the previous instance from `set_property` are unnecessary. `PropertySetGet` uses `MethodBind` to call the setter/getter so the warning will already be printed there.

Perhaps it's necessary when the condition

``` C++
if (!psg->getter)
	return true; //return true but do nothing
```

is true, but I don't know why it returns true instead of false (shouldn't it be an error?).

Reply:
So deprecating a property should just mean deprecating the getter and setter functions of the property?

Reply:
Hmm. Is it possible that one may want to deprecate a property but not its setter and getter? If so, then this may be ok.